src/app/public/src/modules/test-button/test-button.component.html

```diff
@@ -0,0 +1 @@
+<button class="test-button-component" (click)="openWidgetToHelpKey()">Click To Open</button>
```

Comment:
This button is just for testing the key component and the service, correct?

Reply:
It's mostly for demo purposes yes.  Likely will not be in the actual release as it isn't needed